sdk/dapp-kit/test/test-utils.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SuiClient } from '@mysten/sui.js/client';
```

Comment:
shouldn't this be caught by the `consistent-type-imports` eslint rule?

Reply:
it should... it looks like its not enabled for the test directory, I'll get that fixed!